herddb-core/src/main/java/herddb/cluster/BookkeeperCommitLog.java

```diff
@@ -724,6 +731,11 @@ void ensureOpenReader(LogSequenceNumber currentPosition) throws org.apache.bookk
             }
             currentLedger = bookKeeper.openLedgerNoRecovery(ledgerToTail,
                     BookKeeper.DigestType.CRC32C, SHARED_SECRET.getBytes(StandardCharsets.UTF_8));
+            
+            if (LOGGER.isLoggable(Level.FINE)) {
+                String ledgerLeader = extractLeaderFromMetadata(currentLedger.getLedgerMetadata().getCustomMetadata());
```

Comment:
Is this an heavy operation? We don't spam ledger opening.. we could log it even at info level

Reply:
changed to INFO